=== RUN   TestL05_LeakOnNilChan
panic: test timed out after 1m0s
	running tests:
		TestL05_LeakOnNilChan (1m0s)

goroutine 5 [running]:
testing.(*M).startAlarm.func1()
	/home/ilian/Tools/go-runtime/src/testing/testing.go:2484 +0x394
created by time.goFunc
	/home/ilian/Tools/go-runtime/src/time/sleep.go:215 +0x2d

goroutine 1 [chan receive]:
testing.(*T).Run(0xc000102540, {0x57c723?, 0xc000120b30?}, 0x585c18)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1859 +0x431
testing.runTests.func1(0xc000102540)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:2279 +0x37
testing.tRunner(0xc000102540, 0xc000120c70)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1792 +0xf4
testing.runTests(0xc000162000, {0x6ab440, 0x1a, 0x1a}, {0x6b0fc0?, 0x7?, 0x6afe40?})
	/home/ilian/Tools/go-runtime/src/testing/testing.go:2277 +0x4b4
testing.(*M).Run(0xc000124140)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:2142 +0x64a
main.main()
	_testmain.go:95 +0x9b

goroutine 18 [chan send (nil chan)]:
scenarios.TestL05_LeakOnNilChan(0xc000102700?)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_test.go:127 +0x1c
testing.tRunner(0xc000102700, 0x585c18)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1792 +0xf4
created by testing.(*T).Run in goroutine 1
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1851 +0x413
FAIL	scenarios	60.065s
FAIL
FileName:  /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_test.go
TestName:  TestL05_LeakOnNilChan
Import added at line: 3
Header added at line: 125
Header added at file: /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_test.go
go version go1.24.1 linux/amd64 ADVOCATE
=== RUN   TestL05_LeakOnNilChan
SetExitCode: other
panic: runtime error: invalid memory address or nil pointer dereference

goroutine 6 [running]:
runtime.SetExitCodeFromPanicMsg({0x5f1dc0, 0x7418b0})
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/runtime/advocate_exit.go:82 +0x359
runtime.ExitReplayPanic({0x5f1dc0, 0x7418b0})
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/runtime/advocate_replay.go:937 +0x3c
panic({0x5f1dc0, 0x7418b0})
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/runtime/panic.go:744 +0x2f
scenarios.TestL05_LeakOnNilChan(0xc0000e81e0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_test.go:132 +0x50
testing.tRunner(0xc0000e81e0, 0x62b9b0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:1792 +0x1da
created by testing.(*T).Run in goroutine 1
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:1851 +0x8fc

goroutine 1 [chan receive]:
testing.(*T).Run(0xc0000e81e0, {0x621ca8, 0x15}, 0x62b9b0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:1859 +0x92c
testing.runTests.func1(0xc0000e8000)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:2279 +0xad
testing.tRunner(0xc0000e8000, 0xc0000df918)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:1792 +0x1da
testing.runTests(0xc00000e018, {0x7472c0, 0x1a, 0x1a}, {0xc20aa5df508d4b36, 0x8bb2cc70f3, 0x74c260})
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:2277 +0x5ea
testing.(*M).Run(0xc00003e2c0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:2142 +0xc79
main.main()
	_testmain.go:95 +0xa5

goroutine 7 [runnable]:
advocate.InitTracing.func1()
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/advocate/advocate_tracing.go:71
created by advocate.InitTracing in goroutine 6
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/advocate/advocate_tracing.go:71 +0xd2


Exit Replay with code  3 The program panicked unexpectedly
Exit Message: panic: runtime error: invalid memory address or nil pointer dereference

FAIL	scenarios	0.103s
FAIL
